config/routes.rb

```diff
@@ -3,27 +3,33 @@
 
   root 'home#index'
 
-  resources :cards
-  resources :user_sessions, only: [:new, :create, :destroy]
-  resources :users, only: [:new, :create, :destroy]
-
-  resources :blocks do
-    member do
-      put 'set_as_current'
-      put 'reset_as_current'
-    end
+  scope module: 'home' do
+    resources :user_sessions, only: [:new, :create]
+    resources :users, only: [:new, :create]
+    get 'login' => 'user_sessions#new', :as => :login
+
+    post 'oauth/callback' => 'oauths#callback'
+    get 'oauth/callback' => 'oauths#callback'
+    get 'oauth/:provider' => 'oauths#oauth', as: :auth_at_provider
   end
 
-  put 'review_card' => 'trainer#review_card'
+  scope module: 'dashboard' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.